### ISSUE TITLE
(Suggestion) Estimated Startup time counter

### ISSUE THREAD

**Issue by honbee** [PieKing1215/StartupQoL#3]
I absolutely love this mod, thank you for creating it! There is a feature that I think would be a great addition if you're up to it.

It would be very nice if on the loading screen during start up, there was a count down based on the average amount of time that it takes for a pack to start up. That way you can estimate how much time is left for a pack to start up.

**Comment by PieKing1215** [comment]
Nice idea!

I started working on it and I've managed to come up with a decent way to display a timer in the startup screen.
This method I think is as early in the loading process as I could possibly do it.
The text for the timer becomes visible right before mods start loading, so I think the ~20 seconds of loading before that  shouldn't really matter (note the mod still tracks the total time the game has been running even when the timer isn't visible).

This is what it looks like (I might end up changing the formatting for the timer text):

![gif](https://user-images.githubusercontent.com/13819558/104831750-a516ca80-5859-11eb-8cbf-ec64d41ad7ca.gif)

The only real annoyance is that if there's loading steps that take a long time, the game doesn't render so the timer stays stuck. 
There's not really much I can do about this but having the timer at all is better than nothing.

There's a couple things I still want/need to do, but I'll probably have a new version ready within a couple days.
